.circleci/local_publish_helpers.sh

```diff
@@ -31,7 +31,15 @@ function uploadPkgCli {
         aws --profile=s3-uploader s3 cp amplify-pkg-linux-arm64.tgz s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-arm64-$(echo $hash).tgz
         aws --profile=s3-uploader s3 cp amplify-pkg-linux-x64.tgz s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-x64-$(echo $hash).tgz
 
-        if [ "0" -ne "$(aws s3 ls s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-x64 | egrep -v "amplify-pkg-linux-x64-.*" | wc -l)" ]; then
+        INCORRECT_FILES="$(set -o pipefail && aws --profile=s3-uploader s3 ls s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-x64 | ( egrep -v "amplify-pkg-linux-x64-.*" || true ) | wc -l | xargs)"
+        INCORRECT_PERMISSIONS=$?
+
+        if [ INCORRECT_PERMISSIONS -ne "0" ]; then
+            echo "Insufficient permissions to list s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-x64"
+            exit 1
+        fi
+
+        if [ INCORRECT_FILES -ne "0" ]; then
```

Comment:
does "incorrect" mean "already existing" in this context ?

Reply:
Yes

Reply:
Please give it more descriptive name then.

Reply:
I just added another commit with this change.